### ISSUE TITLE
Use GPU for training with stable model

### ISSUE THREAD

**Issue by tarrade** [tarrade/proj_multilingual_text_classification#12]
Use GCP and see if we need to use new derived image from Tensorflow to have all needed drivers

**Comment by tarrade** [comment]
done.

need:
- custom AI Platform notebook with based GPU based image
- edit the VM to add GPU
the rest should work out of the box ;-)

# setup 1: 
48 vCPUs 
80 GB memory
2105/2105 [==============================] - 11700s 6s/step 
                   - accuracy: 0.9148 - loss: 0.2017 - val_accuracy: 0.9479 - val_loss: 0.2135
execution time: 3:15:19

# setup 2: 
8 vCPUs
730 GB memory
1 GPU NVIDIA Tesla P100
2105/2105 [==============================] - 2698s 1s/step 
- accuracy: 0.9216 - loss: 0.1923 - val_accuracy: 0.9323 - val_loss: 0.1978
execution time: 0:45:17
